packages/devtools_app/lib/src/screens/memory/memory_analyzer.dart

```diff
@@ -331,36 +338,39 @@ Map<String, List<String>> drillIn(
   final matcher = ObjectMatcher((className, fields, value) {
     final key = '$className.${fields.join(".")}';
     result.putIfAbsent(key, () => []);
-    result[key].add(value);
+    result[key]!.add(value);
   });
 
-  for (final ClassReference classRef in references) {
-    if (!matcher.isClassMatched(classRef.name)) continue;
+  for (final ClassReference classRef
+      in references as Iterable<ClassReference>) {
+    if (classRef.name == null || !matcher.isClassMatched(classRef.name))
+      continue;
+
+    final name = classRef.name!;
 
     // Insure instances are realized (not Reference.empty).
     computeInstanceForClassReference(controller, classRef);
-    final HeapGraphClassLive liveClass = classRef.actualClass;
+    final HeapGraphClassLive? liveClass = classRef.actualClass;
 
-    AnalysisReference objectNode;
+    late AnalysisReference objectNode;
     if (createTreeNodes) {
       objectNode = AnalysisReference(
-        '${classRef.name}',
-        countNote: liveClass.instancesCount,
+        '$name',
+        countNote: liveClass!.instancesCount,
       );
       librariesNode.addChild(objectNode);
     }
 
     if (_debugMonitorEnabled) {
-      _debugMonitorClass = _debugMonitorClasses.contains(classRef.name)
-          ? '${classRef.name}'
-          : '';
+      _debugMonitorClass = _debugMonitorClasses.contains(name) ? '$name' : '';
```

Comment:
since name is a string, just write `name` instead of `'$name'`

Reply:
done